watertap/unit_models/reverse_osmosis_0D.py

```diff
@@ -889,12 +882,12 @@ def eq_rejection_phase_comp(b, t, j):
                     1 - (b.permeate_side.properties_mixed[t].conc_mass_phase_comp['Liq', j] /
                          b.feed_side.properties_in[t].conc_mass_phase_comp['Liq', j]))
 
-        @self.Constraint(self.flowsheet().config.time)
-        def eq_over_pressure_ratio(b, t):
-            return (b.feed_side.properties_out[t].pressure ==
-                    b.over_pressure_ratio[t]
-                    * (b.feed_side.properties_out[t].pressure_osm
-                    - b.permeate_side.properties_out[t].pressure_osm))
+        @self.Expression(self.flowsheet().config.time,
```

Comment:
I think it's better to have this as an expression but curious what triggered you to do this.

Reply:
I was having a hard time making this constraint well-scaled. I asked @TimBartholomew and he agreed with you that making this an Expression was a good idea.